README.md

```diff
@@ -17,8 +17,8 @@ Guava's ListenableFuture class
 * Maven
 
 ### Runtime dependencies
-* Java 6 or higher
-* Guava 19.0 or higher
+* Java 8 or higher
+* Guava 21.0 or higher
```

Comment:
This mentions 21 but it is compiled against 26, wouldn't compiling against 21 be better (if we do not use any newer apis) for the case where enforcer plugin ensures minimal versions?

Reply:
I guess we could specify a maven range of versions instead (how do you usually specify that you are compatible with >=21 but want to use >=26 if possible?

Reply:
Nwm, I though guava was going to be backwards compatible from 21 but since this method was removed at a later point using a minimal version dependency doesn't help (since we would also have to document that upgrading is safe).